File: Java/1079.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);

        int N , X;

        N = input.nextInt();

        for ( int i = 1 ; i <= N  ; i++){
            double A, B, C,media;

            A = input.nextDouble();
            B = input.nextDouble();
            C = input.nextDouble();

            media = ((A * 0.2) + (B * 0.3) + (C * 0.5));

            System.out.printf("%.1f\n",media);
        }
         
    }
}

File: Python/1079.py
N = int (input ())
media = 0
for i in range (0,N):
    n1,n2,n3 = list (map(float,input().split()))

    media = (n1*0.2 + n2 *0.3 + n3 * 0.5)

    print(f"{media:.1f}")